src/components/rectificationForm/RectificationForm.tsx

```diff
@@ -66,9 +66,16 @@ const RectificationForm = () => {
     }
   };
 
-  const relations = movedCarFormSelected
-    ? ['driver', 'owner', 'poa-holder']
-    : ['driver', 'owner', 'holder'];
+  const relations =
+    selectedForm === 'parking-fine'
+      ? ['driver', 'owner', 'poa-holder']
+      : ['owner', 'poa-holder'];
+
+  useEffect(() => {
+    if (movedCarFormSelected && vehicleRelation !== 'owner') {
+      setVehicleRelation('owner');
+    }
+  }, []);
```

Comment:
The `useEffect` seems like an inefficient solution to this 🤔 
One other solution I could think of would be to define `relations` before `vehicleRelation` and change current line 33 to:
`const [vehicleRelation, setVehicleRelation] = useState(relations[0]);`
so it always chooses the first element from the array.